images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -111,7 +94,25 @@ Push-Location -Path $sdk.FullName
     "add-ons;addon-google_apis-google-21" `
     "cmake;3.6.4111459" `
     "cmake;3.10.2.4988404" `
-    "patcher;v4"
+    "patcher;v4" `
+    "ndk-bundle"
+
+    # Android NDK root path.
+    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+
+    if(Test-Path $ndk_root){
+    
+        $androidNDKs = Get-ChildItem -Path $ndk_root | Sort-Object -Property Name -Descending | Select-Object -First 1
```

Comment:
I am not sure that we need it. What folder will be chosen here?
I think env variables should point to $ndk_root

Reply:
I think so, thank you. changed this.